samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/mesh/state_binding.c

```diff
@@ -0,0 +1,206 @@
+/* Bluetooth: Mesh Generic OnOff, Generic Level, Lighting & Vendor Models
+ *
+ * Copyright (c) 2018 Vikrant More
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <board.h>
+#include <gpio.h>
+
+#include "common.h"
+#include "ble_mesh.h"
+#include "device_composition.h"
+
+#define MINDIFF 2.25e-308
+
+static float sqrt(float square)
```

Comment:
Is there really no existing helper for this in our libc, etc?

Reply:
For that I've to add math.h which could increase ROM size. Plus we've to enable some features via make menuconfig. 

Again ... this is as it is from past few versions. Then why this sudden requirement pops-up ?

I will repairs such things in next PR. Currently my focus is to PASS PTS test requirements & make things better from algorithm/architecture point of view.